packages/plugin-ext/src/main/browser/style/index.css

```diff
@@ -34,6 +34,11 @@
     flex-direction: column;
 }
 
+.plugin-view-container {
```

Comment:
top level classes should be scoped with `theia-` prefix: https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#css-prefix-global-classes

Reply:
Done